lib/mayaUsd/ufe/UsdTransform3dMayaXformStack.cpp

```diff
@@ -520,7 +520,7 @@ Ufe::ScaleUndoableCommand::Ptr UsdTransform3dMayaXformStack::scaleCmd(double x,
                 UsdGeomXformable xformable(usdSceneItem->prim());
 
                 auto s = xformable.AddScaleOp(UsdGeomXformOp::PrecisionFloat, opSuffix);
-                TF_AXIOM(s);
+                TF_VERIFY(s);
```

Comment:
There are similar case for scale etc, for example line 687

Reply:
Those are actually pivot commands. I've noticed any translation cmd such as translateCmd, translateRotatePivotCmd etc do not have any crash issues.

Reply:
I hate that github won't let you comment on unmodified lines!